src/main/java/Main.java

```diff
@@ -0,0 +1,30 @@
+import java.io.IOException;
```

Comment:
Maybe can consider placing this class into a package? Same applies for DialogBox.java, Launcher.java, etc :)